sidekiq-debounce.gemspec

```diff
@@ -30,6 +30,7 @@ DESC
   spec.add_development_dependency 'bundler', '~> 1.6'
   spec.add_development_dependency 'mock_redis'
   spec.add_development_dependency 'mocha'
-  spec.add_development_dependency 'codeclimate-test-reporter'
+  spec.add_development_dependency "simplecov"
+  spec.add_development_dependency "codeclimate-test-reporter", "~> 1.0.0"
```

Comment:
Could you change these to single quotes?  I think Hound is also broken, since it didn't comment 😅